backend/src/bundles/users/types/current-user.type.ts

```diff
@@ -0,0 +1,7 @@
+type CurrentUser = {
```

Comment:
Maybe, it is better to rename it to `User`, so we can reuse it later for other endpoints?

Reply:
Endpoint is using DTO type and this one is for storing inside fastify request.